### ISSUE TITLE
Debugging: Grey text on orange background is hard to read

### ISSUE THREAD

**Issue by pcgeek86** [akamud/vscode-theme-onedark#60]
**Please provide as much information as possible, usually I need these information to actually start working on an issue, so try to fill as much of these as possible**

## Sample code

```
Start-Sleep -Seconds 10
```

Start the debugger (F5)

## Filename and Language selected

* Using the VSCode-PowerShell extension
* PowerShell language selected
* Mac OS X Sierra
* VSCode 1.12.2

## Atom Original theme Screenshot

N/A

## VSCode theme Screenshot

![screen shot 2017-06-06 at 9 38 43 am](https://user-images.githubusercontent.com/466713/26840881-89667926-4a9c-11e7-9e78-455bac073d66.png)

## Versions used

* VSCode version: 1.12.2
* Theme version: 2.0.1

## Extra information

```
PS /> code --list-extensions
Alexander.theme-sublime-oceanicnext
Nuuf.theme-hackershaze
PKief.material-icon-theme
PeterJausovec.vscode-docker
abusaidm.html-snippets
akamud.vscode-theme-onedark
arcticicestudio.nord-visual-studio-code
aws-scripting-guy.cform
azemoh.one-monokai
christian-kohler.npm-intellisense
dbaeumer.jshint
dlech.chmod
donjayamanne.python
dracula-theme.theme-dracula
eg2.vscode-npm-script
fknop.vscode-npm
gerane.Theme-Monokai-Contrast
giladgray.theme-blueprint
idleberg.hopscotch
jakeboone02.mediawiki
jprestidge.theme-material-theme
mrcrowl.hg
ms-vscode.cpptools
ms-vscode.csharp
ms-vscode.PowerShell
ms-vscode.Theme-TomorrowKit
searKing.preview-vscode
vangware.dark-plus-material
vsmobile.vscode-react-native
zhuangtongfa.Material-theme
```

Cheers,
Trevor Sullivan

**Comment by akamud** [comment]
Thanks for taking the time reporting this. I'll release a fix for this in 2.0.2 soon.

I'll update this issue when it is released.

**Comment by akamud** [comment]
This is now released in 2.0.2.
